getting_started_scenarios/ssm_scenario/SPECIFICATION.md

```diff
@@ -0,0 +1,122 @@
+Please hit Enter
+
+The time of the command invocation is 2024-04-09 14:52:50
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+ Now we will create an SSM OpsItem.
+ SSM OpsItem is a feature provided by the Systems Manager service.
+ It is a type of operational data item that allows you to manage and track various operational issues,
+ events, or tasks within your AWS environment.
+
+ You can create OpsItems to track and manage operational issues as they arise.
+ For example, you could create an OpsItem whenever your application detects a critical error
+ or an anomaly in your infrastructure.
+
+Please hit Enter
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+Now we will update SSM OpsItem oi-c1e5435471e9
+Please hit Enter
+
+Now we will resolve the SSM OpsItem oi-c1e5435471e9
+Please hit Enter
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+Would you like to delete the AWS Systems Manager resources? (y/n)
+
+The AWS Systems Manager resources will not be deleted
+--------------------------------------------------------------------------------
+This concludes the AWS Systems Manager SDK Getting Started scenario.
+--------------------------------------------------------------------------------
+
+   ```
+## SOS Tags
+
+The following table describes the metadata used in this SDK Getting Started Scenario.
+
+
+| action                       | metadata file                | metadata key                            |
+|------------------------------|------------------------------|---------------------------------------- |
+| `createOpsItem`              | ssm_metadata.yaml            | ssm_CreateOpsItem                       |
+| `createMaintenanceWindow`    | ssm_metadata.yaml            | ssm_CreateMainWindow                    |
+| `updateMaintenanceWindow`    | ssm_metadata.yaml            | ssm_UpdateMainWindow                    |
+| `createDocument`             | ssm_metadata.yaml            | ssm_CreateDocument                      |
+| `sendCommand `               | ssm_metadata.yaml            | ssm_SendCommand                         |
+| `commandInvocations`         | ssm_metadata.yaml            | ssm_DescribeInvocation                  |
+| `updateOpsItem`              | ssm_metadata.yaml            | ssm_UpdateOpsItem                       |
+| `deleteMaintenanceWindow `   | ssm_metadata.yaml            | ssm_DeleteMainWindow                    |
+| `deleteDocument`             | ssm_metadata.yaml            | ssm_DeleteMainWindow                    |
+| `describeOpsItems       `    | ssm_metadata.yaml            | ssm_DescribeOpsItems                    |
```

Comment:
Actions should use Rest API casing (typically PascalCase), not language-specific casing.

Reply:
Fixed action names